src/App.js

```diff
@@ -152,29 +154,146 @@ class Board extends React.Component {
     }
   }
 
+  /*
+   * This method checks the solutions for the specified row. 
+   * A row is valid if there's 9 values and all numbers 1-9 are present.
+   * Returns true if valid row, false otherwise
+   *
+   * @param rowToVerify is an integer number, board is a 2D array representing Sudoku board
+   */
+  static verifyRow(rowToVerify, board) {
+    let numCol = 9;
+    let maxSudokuValue = 9;
+    const rowSet = new Set(); //create new set to store values of row
+    
+    //add values of board into Set
+    for (let col = 0; col < numCol; col++) {
+      rowSet.add(board[rowToVerify][col]);
+    }
+
+    //should have 9 values in row
+    if (rowSet.size != maxSudokuValue) {
```

Comment:
what if there was a number in the set greater than 9 or less than 1? For example,

[0, 1, 2, 3, 4, 5, 6, 7, 8] has 9 elements but is not a valid row.

Reply:
Nevermind, I see you check for that below.